.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Add a brief and meaningful description. -->
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
```

Comment:
Hi @JameelB,

Is this env good for this project? Could you help me here by checking this PR?

Reply:
I don't think it's required here because the operator is usually running inside a container with a fixed base image.

Reply:
What env you do not think is required? @pb82

Reply:
Ah sorry, i meant the operating system. Not sure if we need that info in the typical usage scenarios but it also won't hurt.